1.9/administration/upgrading.md

```diff
@@ -18,132 +18,75 @@ This document provides instructions for upgrading a DC/OS cluster from version 1
 ## Prerequisites
 
 - Mesos, Mesos Frameworks, Marathon, Docker and all running tasks in the cluster should be stable and in a known healthy state.
-- For Mesos compatibility reasons, we recommend upgrading any running Marathon-on-Marathon instances to version Marathon 0.16.0-RC3 before proceeding with this DC/OS upgrade.
-- You must have access to copies of the config files used with DC/OS 1.7: `config.yaml` and `ip-detect`.
+- For Mesos compatibility reasons, we recommend upgrading any running Marathon-on-Marathon instances to version Marathon 1.3.5 before proceeding with this DC/OS upgrade.
```

Comment:
Is this version intended to match the Marathon we're shipping with 1.9? Looks like we're shipping 1.4.0: https://github.com/dcos/dcos/blob/master/packages/marathon/buildinfo.json#L5